drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -171,8 +171,10 @@ Error RasterizerGLES2::is_viable() {
 // GLVersion seems to be used for both GL and GL ES, so we need different version checks for them
 #ifdef OPENGL_ENABLED // OpenGL 2.1 Profile required
 	if (GLVersion.major < 2 || (GLVersion.major == 2 && GLVersion.minor < 1)) {
+		ERR_PRINT("Error OpenGL Version < 2.1");
```

Comment:
`ERR_PRINT` already adds `ERROR: `, so no need to restate it.
Would be good to give more info too since we have it, e.g.:
```
ERR_PRINTS(vformat("This system only supports OpenGL %d.%d, while OpenGL 2.1 is required.", GLVersion.major, GLVersion.minor));
```
Same below for OpenGL ES 2.0.

Reply:
Bump.

Reply:
@akien-mga #26987 touches same code and makes this somewhat obsolete, so I didn't update it. Better close and archived this.